src/AzureClient/AzureClient.csproj

```diff
@@ -6,6 +6,7 @@
     <RootNamespace>Microsoft.Quantum.IQSharp.AzureClient</RootNamespace>
     <AssemblyName>Microsoft.Quantum.IQSharp.AzureClient</AssemblyName>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	<LangVersion>10.0</LangVersion>
```

Comment:
Very minor nit, but spacing here is inconsistent (1 hard tab versus four spaces).

Reply:
fixed